projects/rocprofiler-compute/CMakeLists.txt

```diff
@@ -41,6 +41,74 @@ set(EXECUTABLE_NAME "rocprof-compute")
 include(ExternalProject)
 include(GNUInstallDirs)
 
+find_program(HIPCC_EXECUTABLE hipcc)
+if(NOT HIPCC_EXECUTABLE)
+    message(
+        WARNING
+            "hipcc not found! Skipping kokkos and kokkos-miniapps build. The rest of the build will proceed."
+        )
+    set(ENABLE_KOKKOS OFF)
+else()
+    set(ENABLE_KOKKOS ON)
+endif()
+
+if(ENABLE_KOKKOS)
+    # Detect AMD GPU architecture using rocminfo
+    execute_process(
+        COMMAND bash -c
+                "rocminfo | grep -E 'Name: gfx[0-9]+' | head -n 1 | awk '{print \$2}'"
```

Comment:
It seems not a reliable solution for me. How to handle (a) if cannot find rocminfo, (b) awk won't work for WIN build in the future.